### ISSUE TITLE
Version 1.19.0 - Failed to generate localization files 

### ISSUE THREAD

**Issue by pro100andrey** [localizely/flutter-intl-vscode#96]
Hello,

I have generation issue after update to version 1.19.0:

`Failed to generate localization files for the "localization" project.`

output log:

`INFO: Flutter Intl extension enabled for the 'localization' project.`

Version  1.18.2 work fine

**Comment by lzoran** [comment]
Can't reproduce this issue. :/
Could you provide us more details:
- OS
- VS Code version
- Flutter version
- Do you use the `intl_utils` package as a dependency (or dev dependency) in your `pubspec.yaml` file, and if you use which version?
- Could you provide us a minimal reproducible sample or check if the Flutter Intl extension works with the [sample app](https://github.com/localizely/flutter-intl-plugin-sample-app)?

<br/>

Could you also check if reactivation of the `intl_utils` package helps until we find the cause of the problem?
  - Step 1. `flutter pub global deactivate intl_utils`
  - Step 2. Trigger code generation (e.g. update and save one of the arb files)

**Comment by khobotin-elopage** [comment]
Same issue here. I think, VS Code plugin auto-updated to v.1.19. And now I'm getting this error.

- macOs Ventura 13.0.1
- VS Code v.1.73.1
- Flutter v.3.3.8
- Dont use `intl_utils`

**Comment by pro100andrey** [comment]
Hi @lzoran, thanks for quick response!

macOs Ventura 13.0.1
VS Code v.1.73.1
Flutter v3.3.9
Dont use intl_utils

Minimal reproducible sample:
[localization.zip](https://github.com/localizely/flutter-intl-vscode/files/10083827/localization.zip)

**Comment by khobotin-elopage** [comment]
As the temporary solution, we can add [intl_utils](https://pub.dev/packages/intl_utils) dependency and run `flutter pub run intl_utils:generate`

**Comment by pro100andrey** [comment]
@khobotin-elopage or switch to previous version:

<img width="694" alt="Screenshot 2022-11-24 at 13 13 12" src="https://user-images.githubusercontent.com/46396672/203770563-92fb1006-6cd4-42f5-b57f-309668ae3e20.png">

Thanks!

**Comment by lzoran** [comment]
I think I found what is the cause.
It seems that extension tries to access `dev_dependencies` in the `pubspec.yaml` file and fails with generation if missing.

We will try to release a fix for this soon.
Until we fix that, some workarounds:
- Use empty `dev_dependencies`
    ```
    dev_dependencies: []
    ```
- Use `dev_dependencies` with `intl_utils` dependency 
    ```
    dev_dependencies:
      intl_utils: ^2.8.1
    ```
- Switch to the previous version of the Flutter Intl extension

**Comment by pro100andrey** [comment]
@lzoran Thank you!

**Comment by okdevy** [comment]
HI. Having same issue. Flutter Intl v1.19.0
~ flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.3.8, on macOS 13.0.1 22A400 darwin-arm, locale
    en-UA)
[✓] VS Code (version 1.73.1)


UPD: Solved. Thanks!

**Comment by lzoran** [comment]
This should be fixed with the `Flutter Intl 1.19.1`.

I'm closing this issue as resolved.
In case you notice any irregularities, feel free to open it again.
